aws/app/lambda/submission/submission.js

```diff
@@ -84,7 +85,18 @@ const saveData = async (submissionID, formData) => {
 
     const formSubmission = typeof formData === "string" ? formData : JSON.stringify(formData);
     const timeStamp = Date.now().toString();
-    const DBParams = {
+
+    const formSubmissionAsString = typeof formData.responses === "string" ? formData.responses : JSON.stringify(formData.responses);
+    const formSubmissionAsHash = crypto.createHash("md5").update(formSubmissionAsString).digest("hex");
```

Comment:
Curious as to why md5 instead of SHA256?

Reply:
I do remember mentioning something to you about SHA256 and the fact that it was better for our use case but after doing some more research I figured we would be perfectly fine using  MD5. Mainly because it is faster to calculate a MD5 hash compared to SHA256.

Reply:
Ok, let's add a comment in the code that we're using MD5 because of it's speed and we are not using it for cryptography but only for checksum validation.